appserver/tests/application/src/main/java/org/glassfish/main/test/app/security/jakartasecurity/CustomPrincipal.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2015, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Are some files in this PR resurrected or new?

Reply:
You're right. I think I copied some older example and modified it. I fixed the copyright headers in all files now.